<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValueConfigDlg</class>
 <widget class="QDialog" name="ValueConfigDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Value Config</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Overrides</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Logical Min</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="txtLogicalMinOverride"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Device:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="lblLogicalMinDefault">
          <property name="text">
           <string>-00000</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Logical Max</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Device:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="lblLogicalMaxDefault">
          <property name="text">
           <string>000000</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="txtLogicalMaxOverride"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="cboType"/>
        </item>
        <item row="0" column="4">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Output Type:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cboOutputType">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Min:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtMinOutput">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Max:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtMaxOutput">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="chkReverse">
          <property name="text">
           <string>Reverse Axis</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Response</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="chkUseCPoints">
        <property name="text">
         <string>Use Response Curve</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ControlPointsWidget" name="cpointView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="chkSnapToGrid">
          <property name="text">
           <string>Snap To Grid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblCPointPos">
          <property name="text">
           <string>(0,0)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCPoint1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCPoint2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCPoint3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCPoint4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCPointCustom">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Custom</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ControlPointsWidget</class>
   <extends>QGraphicsView</extends>
   <header>controlpointswidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ValueConfigDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>633</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ValueConfigDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>633</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cboOutputType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ValueConfigDlg</receiver>
   <slot>onOutputTypeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>639</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkSnapToGrid</sender>
   <signal>clicked(bool)</signal>
   <receiver>ValueConfigDlg</receiver>
   <slot>onSnapToGridClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>73</x>
     <y>638</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCPoint1</sender>
   <signal>clicked()</signal>
   <receiver>ValueConfigDlg</receiver>
   <slot>onCPoint1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>580</y>
    </hint>
    <hint type="destinationlabel">
     <x>751</x>
     <y>574</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCPoint2</sender>
   <signal>clicked()</signal>
   <receiver>ValueConfigDlg</receiver>
   <slot>onCPoint2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>749</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCPoint3</sender>
   <signal>clicked()</signal>
   <receiver>ValueConfigDlg</receiver>
   <slot>onCPoint3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>614</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>747</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCPoint4</sender>
   <signal>clicked()</signal>
   <receiver>ValueConfigDlg</receiver>
   <slot>onCPoint4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>651</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>748</x>
     <y>601</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCPointCustom</sender>
   <signal>clicked()</signal>
   <receiver>ValueConfigDlg</receiver>
   <slot>onCPointCustom()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>702</x>
     <y>581</y>
    </hint>
    <hint type="destinationlabel">
     <x>750</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtLogicalMinOverride</sender>
   <signal>editingFinished()</signal>
   <receiver>ValueConfigDlg</receiver>
   <slot>onLogicalMinEditFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtLogicalMaxOverride</sender>
   <signal>editingFinished()</signal>
   <receiver>ValueConfigDlg</receiver>
   <slot>onLogicalMaxEditFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>329</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUseCPoints</sender>
   <signal>clicked(bool)</signal>
   <receiver>ValueConfigDlg</receiver>
   <slot>onUseCPointsClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkReverse</sender>
   <signal>clicked(bool)</signal>
   <receiver>ValueConfigDlg</receiver>
   <slot>onReverseAxisClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>699</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>748</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOutputTypeChanged(int)</slot>
  <slot>onSnapToGridClicked(bool)</slot>
  <slot>onCPoint1()</slot>
  <slot>onCPoint2()</slot>
  <slot>onCPoint3()</slot>
  <slot>onCPoint4()</slot>
  <slot>onCPointCustom()</slot>
  <slot>onLogicalMinEditFinished()</slot>
  <slot>onLogicalMaxEditFinished()</slot>
  <slot>onUseCPointsClicked(bool)</slot>
  <slot>onReverseAxisClicked(bool)</slot>
 </slots>
</ui>
